mobile_back_tflite/cpp/backend_tflite/tflite_settings_android.h

```diff
@@ -96,6 +96,7 @@ benchmark_setting {
   accelerator_desc: "NNAPI"
   configuration: "TFLite"
   src: "https://github.com/mlcommons/mobile_models/raw/main/v0_7/tflite/mobilenet_edgetpu_224_1.0_uint8.tflite"
+  single_stream_expected_latency_ns: 900000
```

Comment:
The vendor can change the default value of 1M by explicitly set it in the `benchmark_setting`